src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -223,6 +227,10 @@ trait WorkflowComponent {
       filter(_.id === id)
     }
 
+    def findWorkflowByExternalIdAndSubmissionId(externalId: String, submissionId: UUID): WorkflowQueryType = {
```

Comment:
we used to use cromwell's workflow id as our primary key but that caused unit test problems and is a bad design for the future so there is now an external id which is cromwell's workflow id and our own numeric PK